ansible/roles/epel-repo-install/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+---
+# epel_repo_install tasks
+
+- name: RHEL tasks
+  block:
+    - name: install EPEL repo
+      ansible.builtin.dnf:
+        name: 'https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm'
+        disable_gpg_check: true
+        state: present
+  when: ansible_distribution == "RedHat"
+
+- name: CentOS tasks
+  block:
+    - name: enable powertools repo (CentOS 8)
+      ansible.builtin.command: dnf config-manager --set-enabled powertools
+      when: ansible_distribution_version == "8"
+
+    # There's no CRI-O for RHEL/CentOS 9
```

Comment:
We do not support RHEL 9 now, so can we remove this?

Reply:
This is skipped on rhel9:
```
TASK [install-logging : enable powertools repo (CentOS 9)] ******************************************************************************************************************
skipping: [logging.example.com] => {"changed": false, "skip_reason": "Conditional result was False"}
```

Reply:
It would be skipped on RHEL9 as it's a task for CentOS hosts.
I will change the comment so I don't mention RHEL.

Reply:
Yes, I think let's remove this. Otherwise, it's confusing.

Reply:
Removed